Diffusion-macOS/ControlsView.swift

```diff
@@ -242,7 +248,11 @@ struct ControlsView: View {
                             Text("Guidance Scale")
                             Spacer()
                             Text(guidanceScaleValue)
-                        }.padding(.leading, 10)
+                        }
+                        .onChange(of: generation.guidanceScale) { guidanceScale in
+                            Settings.shared.guidanceScale = guidanceScale
+                        }
```

Comment:
Yes, this makes sense to me

Reply:
Also we may want to add a "default" value per model, since SD3 prefers smaller cfg values.

Reply:
But then when would we use it? If the user has modified the guidance scale in a model then uses a different one with another default, it would still use the value stored in the settings, wouldn't it? Let's do it in a followup PR, unless you have a clear understanding of how to do it :)

Reply:
It would likely have to be a default upon switching models, but I agree we can follow up.